src/cdk/overlay/public_api.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {NgModule, Provider} from '@angular/core';
+import {PortalModule} from '@angular/cdk/portal';
+import {Overlay} from './overlay';
+import {ScrollDispatchModule} from './scroll/index';
+import {
+  ConnectedOverlayDirective,
+  OverlayOrigin,
+  MD_CONNECTED_OVERLAY_SCROLL_STRATEGY_PROVIDER,
+} from './overlay-directives';
+import {OverlayPositionBuilder} from './position/overlay-position-builder';
+import {VIEWPORT_RULER_PROVIDER} from './position/viewport-ruler';
+import {OVERLAY_CONTAINER_PROVIDER} from './overlay-container';
+
+
+export {VIEWPORT_RULER_PROVIDER} from './position/viewport-ruler';
```

Comment:
It might be a good idea to move these exports down after the `NgModule`. I've found myself mistaking them for `import` statements on a few occasions.